marklogic-data-hub-central/ui/src/pages/Load.tsx

```diff
@@ -10,7 +10,7 @@ import { AuthoritiesContext } from "../util/authorities";
 
 export type ViewType =  'card' | 'list';
 
-const INITIAL_VIEW: ViewType = 'list';
+const INITIAL_VIEW: ViewType = 'card';
```

Comment:
Can you pls verify Load.test if it needs some changes. It sets default view to be list and asserts the same.